spec/lib/authorities/linked_data/term_config_spec.rb

```diff
@@ -72,12 +72,12 @@
         term_id: 'ID',
         language: ['es'],
         results: {
-          id_predicate: 'http://purl.org/dc/terms/identifier',
-          label_predicate: 'http://www.w3.org/2004/02/skos/core#prefLabel',
-          altlabel_predicate: 'http://www.w3.org/2004/02/skos/core#altLabel',
-          broader_predicate: 'http://www.w3.org/2004/02/skos/core#broader',
-          narrower_predicate: 'http://www.w3.org/2004/02/skos/core#narrower',
-          sameas_predicate: 'http://www.w3.org/2004/02/skos/core#exactMatch'
+          id_ldpath:       'dcterms:identifier',
+          label_ldpath:    'skos:prefLabel',
```

Comment:
This is the primary change for the PR. The config now specifies results as ldpaths instead of predicates.